vkconfig/mainwindow.cpp

```diff
@@ -186,7 +186,9 @@ void MainWindow::LoadConfigurationList() {
         item->configuration = configurator._available_configurations[i];
         ui->profileTree->addTopLevelItem(item);
         item->radio_button = new QRadioButton();
+#ifndef _WIN32 // This is a hack to workaround a Linux layout issue
         item->radio_button->setText("-");
```

Comment:
Removing the - is not as simple as this. We still need some spaces, either in the radio button text, or prepended to the description in the next column. Roll this back, and I'll make the changes and test it on all platforms under a separate commit.

Reply:
I am not seeing any layout issue on Windows with my change but feel free to improve my change with a future PR. 

![image](https://user-images.githubusercontent.com/62888873/90655423-d440f580-e241-11ea-80ae-8651686b195b.png)

Reply:
Yeah, looks fine on Windows... Okay, go ahead and do this and I'll fix it after the fact for the other platforms.